app/models/pageviews/RegistrationConfirmationViewModel.scala

```diff
@@ -22,51 +22,64 @@ import models.subscription.{Contact, IndividualContact, OrganisationContact}
 import models.{Mode, UserAnswers}
 import pages.*
 import play.api.data.Form
+import play.api.i18n.{Lang, Messages}
+import utils.DateTimeFormats.{dateFormat, timeFormat}
+import java.time.ZoneId
 
 case class RegistrationConfirmationViewModel(
                                               mode: Mode,
                                               form: Form[Boolean],
                                               dprsUserId: String,
+                                              subscribedDate: String,
+                                              subscribedTime: String,
```

Comment:
You don't need two values here. You can format the string with the following formatter:
DateTimeFormatter.ofPattern("d MMMM yyyy 'at' hh:mma (z)").withZone(ZoneId.of("GMT"))